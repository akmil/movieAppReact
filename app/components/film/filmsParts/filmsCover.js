import React from 'react';

export const FilmsCover = ({ src }) => (
    <img src={src}
         alt={src}
         className='col-33 mr-3'
    />
);

// default FilmsCover;
