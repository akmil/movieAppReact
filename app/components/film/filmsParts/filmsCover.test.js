import React from 'react';
// import { shallow } from 'enzyme';
import {FilmsCover} from './filmsCover';
import renderer from 'react-test-renderer';

describe('<films FilmsCover />', () => {
    it('should render SearchButton without errors', () => {
        const wrapper = renderer.create(<FilmsCover/>);
        // const wrapper = shallow(<MainLayout/>);

        expect(wrapper).toMatchSnapshot();
    })
});