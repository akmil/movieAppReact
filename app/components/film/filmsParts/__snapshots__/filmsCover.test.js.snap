// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<films FilmsCover /> should render SearchButton without errors 1`] = `
<img
  alt={undefined}
  className="col-33 mr-3"
  src={undefined}
/>
`;
